app/scripts/templates/complete_account_unlock.mustache

```diff
@@ -0,0 +1,37 @@
+<div id="main-content">
+  {{#isLinkExpired}}
+  <header>
+    <h1 id="fxa-account-unlock-link-expired-header">{{#t}}Unlock link expired{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+
+    <p>
+      {{#t}}The link you clicked to verify your email is expired.{{/t}}
```

Comment:
I'm not a wordsmith, but should this be "email _is_ expired" or "email _has_ expired"?
/cc @ryanfeeley @johngruen